CMakeLists.txt

```diff
@@ -236,6 +236,7 @@ set(ARANGO_EXPORT_FRIENDLY_STRING  "arangoexport - data exporter")
 set(ARANGO_IMPORT_FRIENDLY_STRING  "arangoimport - data importer")
 set(ARANGOSH_FRIENDLY_STRING       "arangosh - commandline client")
 set(ARANGO_VPACK_FRIENDLY_STRING   "arangovpack - VelocyPack formatter")
```

Comment:
DRY please. ARANGODB_PACKAGE_VENDOR can be used.

Reply:
Done